addons/website_slides/static/src/js/slides_upload_toast.js

```diff
@@ -0,0 +1,32 @@
+odoo.define('website_slides.upload_toast', function (require) {
+    'use strict';
+    
+    var publicWidget = require('web.public.widget');
+
+    var sessionStorage = window.sessionStorage;
+    var core = require('web.core');
+    var _t = core._t;
+    
+    
+    publicWidget.registry.UploadToast = publicWidget.Widget.extend({
+        selector: '.o_upload_toast',
+        
+        /**
+         * @private
+         */
+        start: function () {
+            this._super.apply(this, arguments);
+            var url = sessionStorage.getItem("toast");
+            if (url) {
+                var message = _.str.sprintf(_t('Follow this link to add questions to your certification. <a href="%s">Edit certification</a>'), url);
+                this.displayNotification({
+                    type: 'info',
+                    title: _t('Certification created'),
+                    message: message,
+                    sticky: false
+                });
+                sessionStorage.removeItem("toast");
+            }
```

Comment:
as `start` is a deferred, you should wait that super is ready before doing your stuff.
and you should return the all thing
do this instead : 
```
start: function () {
    return this._super.apply(this, arguments).then(function () {
        // yourStuff
    });
},
```